### DISCUSSION TITLE
Light/dark theme selector

### DISCUSSION THREAD

**Discussion by LeaVerou** [shoelace-style/webawesome#942] [Ideas & Suggestions]
This is a very common component, especially for docs, and one we can get for cheap because we need to implement one for our own needs *anyway*.

## Examples

WA:
<img width="148" alt="image" src="https://github.com/user-attachments/assets/c4271ba7-8919-4765-a558-a94a9040d92f">

LitElement:
<img width="223" alt="image" src="https://github.com/user-attachments/assets/8ea26ae1-729e-4f07-bdeb-94f34519ecf3">

ESLint:
<img width="253" alt="image" src="https://github.com/user-attachments/assets/e7318b15-572d-4c4a-b98e-ab7561022f3e">

X:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/5bd256c9-a8db-4aef-98cc-0b5e6fe12f08" />
<img width="592" alt="image" src="https://github.com/user-attachments/assets/a5283377-7e64-4f84-b686-e3b3e551a562" />

SuperColorPalette: 

<img width="112" alt="Image" src="https://github.com/user-attachments/assets/50d4bec2-7586-4407-87ec-782d303ec1d1" />

**Comment by ogrodnek** [comment]
i accidentally made a dupe of this (https://github.com/shoelace-style/webawesome/discussions/1507)

would really like this, especially if it could save/read selections..

i've used https://www.webcomponents.org/element/dark-mode-toggle

before but obviously it would be great to have something more WA specific/integrated

**Comment by ogrodnek** [comment]
i accidentally made a dupe of this (https://github.com/shoelace-style/webawesome/discussions/1507)

would really like this, especially if it could save/read selections..

i've used https://www.webcomponents.org/element/dark-mode-toggle

before but obviously it would be great to have something more WA specific/integrated
